### PR TITLE
Feature/tc 56

### PR DESCRIPTION
**Pull request by CcConStanTine**

### CODE REVIEW

File: src/main/java/codeenthusiast/TrainingCenterApp/exercise/strengthexercise/keytechniqueelement/KeyTechniqueController.java [hunk 1]
```diff
@@ -0,0 +1,20 @@
+package codeenthusiast.TrainingCenterApp.exercise.strengthexercise.keytechniqueelement;
+
+import codeenthusiast.TrainingCenterApp.abstracts.AbstractController;
+import codeenthusiast.TrainingCenterApp.abstracts.AbstractService;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+public class KeyTechniqueController extends AbstractController<KeyTechniqueElement, KeyTechniqueElementDTO> {
```
Comment (naslakboss) [inline src/main/java/codeenthusiast/TrainingCenterApp/exercise/strengthexercise/keytechniqueelement/KeyTechniqueController.java:8]: You should stick to one version of KeyTechnique : KeyTechnique or KeyTechniqueElements

File: src/test/java/codeenthusiast/TrainingCenterApp/exercise/strengthexercise/keytechniqueelement/KeyTechniqueElementServiceImplTest.java [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+
+    KeyTechniqueElement keyTechniqueElement1;
+    KeyTechniqueElement keyTechniqueElement2;
+    KeyTechniqueElement keyTechniqueElement3;
+
+    @BeforeEach
+    void setUp() {
+        keyTechniqueElement1 = new KeyTechniqueElement("Rozstaw nogi na szerkości barków");
+        keyTechniqueElement2 = new KeyTechniqueElement("Stopy delikatnie na zewnątrz");
+        keyTechniqueElement3 = new KeyTechniqueElement("Napnij mięsnie brzucha");
```
Comment (naslakboss) [inline src/test/java/codeenthusiast/TrainingCenterApp/exercise/strengthexercise/keytechniqueelement/KeyTechniqueElementServiceImplTest.java:22]: We should choose one convention and use only english or only polish in test data fragments of code

**Review by naslakboss — CHANGES_REQUESTED** [review]
Wonderful abstract code, chlop.. I saw only two small naming convention

**Review by CcConStanTine — COMMENTED** [review]
Understood.

**Review by CcConStanTine — COMMENTED** [review]
Understood

**Review by naslakboss — APPROVED** [review]

**Pull request merged**